vlsi/example-designs/sky130-openroad.yml

```diff
@@ -0,0 +1,152 @@
+# Override configurations in ../example-sky130.yml
+
+# Specify clock signals
+vlsi.inputs.clocks: [
+  {name: "clock_clock", period: "10ns", uncertainty: "1ns"}
+]
+
+# # Power Straps
```

Comment:
Looks like the openroad overrides the floorplan size and clock period to relax it (for obvious reasons)? Maybe remove all these commented constraints and note in the documentation why these are overridden.